cli/src/main/java/com/devonfw/tools/ide/tool/java/JavaUrlUpdater.java

```diff
@@ -16,6 +16,19 @@ protected String getTool() {
     return "java";
   }
 
+  @Override
+  protected String mapVersion(String version) {
+
+    // remove the suffix "-LTS", this is necessary for java version 21+35-LTS
+    if (version.length() > 4) {
+      String lastFourChars = version.substring(version.length() - 4);
+      if (lastFourChars.equals("-LTS")) {
```

Comment:
while this is technically correct, Java allows to do such things more readable by `version.endsWith("-LTS")`.